sampling/hierarchical/opentelemetry-sampling-hierarchical.gemspec

```
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Copyright OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require('opentelemetry/sampling/hierarchical/version')
+
+Gem::Specification.new do |spec|
+  spec.name = 'opentelemetry-sampling-hierarchical'
+  spec.version = OpenTelemetry::Sampling::Hierarchical::VERSION
+  spec.authors = ['OpenTelemetry Authors']
+  spec.email = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary = 'Hierarchical Sampling Extension for the OpenTelemetry framework'
+  spec.description = 'Hierarchical Sampling Extension for the OpenTelemetry framework'
+  spec.homepage = 'https://github.com/open-telemetry/opentelemetry-ruby-contrib'
+  spec.license = 'Apache-2.0'
+
+  spec.files = Dir.glob('lib/**/*.rb') + Dir.glob('*.md') + %w[LICENSE .yardopts]
+  spec.require_paths = %w[lib]
+  spec.required_ruby_version = '>= 2.6.0'
```

We do not support versions of EOL Ruby versions and target Ruby 3.0 syntax.

```suggestion
  spec.required_ruby_version = '>= 3.0.0'
```